filter.js

```diff
@@ -133,10 +141,10 @@ const sentEventClick = (eventName) => {
     };
 
     const displayPriceWithCurrency = (singleEvent) => {
-      if (Number.isFinite(singleEvent.cost)) {
+      if (singleEvent.cost > 0) {
```

Comment:
nitpick: realnie można skrócić i ominąć complexity trochę : 

if(singleEvent.cost){
    return 'Darmowe'; 
}
return `${singleEvent.cost.toFixed(2).replace(/\./g, ",")} zł`


+ nie korzystasz z parametrów to bez odwróconych ciapek